lib/moonshot/controller_config.rb

```diff
@@ -21,5 +25,17 @@ def initialize
       @plugins = []
       @show_all_stack_events = false
     end
+
+    def parameter_strategy=(value)
```

Comment:
We talked about putting this case statement on the CLI side, I'm wondering why you ended up putting it here instead? One reason I like it on the CLI side is that it's possible for advanced users to make a new wrapper around the `Moonshot::Controller` to manage their infrastructure. If this simply accepted a class that behaved as a parameter strategy, a user could bring their own parameter strategy and run with it. With it baked into the controller config, they'd have to resort to monkeypatching this class or getting their changes merged upstream.